### PR TITLE
chore(limits): update resource limits and replica count for prod

### PR DESCRIPTION
**Pull request by lgarofalo**

### CODE REVIEW

**Comment by lgarofalo** [comment]
Staging has been running with these changes for a few months.

File: production/k8s-askdarcel.yaml [hunk 1]
```diff
@@ -74,7 +81,7 @@ metadata:
    app: web
  name: web
 spec:
- type: LoadBalancer
+ type: NodePort
```
Comment (richardxia) [inline production/k8s-askdarcel.yaml]: Is there a reason we're switching this from the LoadBalancer service type to the NodePort type? I'm not specifically familiar with GCloud, but the LoadBalancer specific to each cloud vendor should be able to trivially handle two replicas. Most guides I'm reading online seem to suggest not using the NodePort type.

Or is it due to cost, and we don't want to pay for a dedicated LoadBalancer?

Comment (lgarofalo) [reply production/k8s-askdarcel.yaml]: We have an ingress defined above which handles exposing the service externally, so the LoadBalancer was redundant.
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: ingress
spec:
  tls:
  - secretName: origin-cert-secret
  backend:
    serviceName: web
    servicePort: 80
```
I did this out of familiarity originally.  The ingress gives some more flexibility compared to a LoadBalancer, but I'm open to changing it if we have reason.

**Comment by lgarofalo** [comment]
I'll merge this after deployment is successful.

**Pull request merged**